lib/driver.js

```diff
@@ -312,6 +308,17 @@ class XCUITestDriver extends BaseDriver {
       await this.opts.device.clearCaches('com.apple.mobile.installd.staging');
 
       await this.startSim();
+
+      if (this.opts.customSSLCert) {
+        log.info(`Installing ssl cert ${_.truncate(this.opts.customSSLCert, 20)}`);
+        log.warn(`If a certificate from this domain was already installed this test may fail and can be fixed by
+          erasing the contents of the Simulator (Simulator > Hardware > Erase All Content and Settings) and running the tests again`);
+        await installSSLCert(this.opts.customSSLCert, this.opts.udid);
+        await this.opts.device.shutdown();
```

Comment:
Simulator restart is an expensive operation. We should not restart it every time when the corresponding capability matches, but rather do it once, change the necessary stuff and start it when all the changes are completed.

Reply:
also, is there a way to check the certificate is already there to avoid undesired restarts for each session?